src/backend/tests/teams.test.ts

```diff
@@ -117,4 +117,40 @@ describe('Teams', () => {
       expect(prisma.team.findUnique).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('setTeamHead', () => {
+    test('setTeamHead head not found', async () => {
+      jest.spyOn(prisma.team, 'findUnique').mockResolvedValue(prismaTeam1);
+
+      const callSetTeamHead = async () => await TeamsService.setTeamHead(flash, sharedTeam1.teamId, 122);
+
+      const expectedException = new HttpException(404, 'User with id: 122 not found!');
+
+      await expect(callSetTeamHead).rejects.toThrow(expectedException);
+    });
+
+    test('setTeamHead works', async () => {
```

Comment:
can you test the team not found and the new is not head validation cases as well